### ISSUE TITLE
eventType not working

### ISSUE THREAD

**Issue by txs** [JackAdams/meteor-editable-text#52]
I followed the doc and mousein doesn't work and the mouseover or mouseenter don't work either!
Hope to fix this!

**Comment by JackAdams** [comment]
Thanks for reporting.  There's definitely something going wrong -- I'll look into it.

**Comment by JackAdams** [comment]
Unfortunately, this needs a pretty substantial refactor to get it going properly.  For the meantime, it's not going to support any events except `"click"`, `"dblclick"`, `"mousedown"`.  I've changed the docs to reflect this.

**Comment by txs** [comment]
It's ok I use the meteor events to sent the fake one. However, you should
change the doc to tell people not to use trigger. it triggers too many
times. However, without trigger seams fine.

2015-09-24 7:48 GMT+08:00 Brent Abrahams notifications@github.com:

**Comment by JackAdams** [comment]
Yes, you're right -- the `editable-text-trigger` does cause recursion for any other event types.

**Comment by txs** [comment]
please change the readme

2015-09-25 6:59 GMT+08:00 Brent Abrahams notifications@github.com:

**Comment by JackAdams** [comment]
Added a warning to the README.  Leaving this issue open as those events like mouseover, mouseenter should really be supported as well.  It's just not going to happen anytime in the near future, unfortunately.

**Comment by txs** [comment]
No problem. I did a hack myself :D

2015-10-05 21:59 GMT+08:00 Brent Abrahams notifications@github.com:
